tests/unittests/graphOptzTest.cpp

```diff
@@ -1818,17 +1776,17 @@ TEST_F(GraphOptz, VarsCSE) {
   auto *SN = F_->createSigmoid("sigmoid", input2);
   auto *RN = F_->createRELU("relu", input3);
   auto *CN = F_->createConcat("concat", {TN, SN, RN}, /* axis */ 0);
-  F_->createSave("ret", CN);
+  F_->createSave(ctx_, "ret", CN);
```

Comment:
mod_.createVariable was not changed.

Reply:
Right - there are several tests in graphOptz that rely on a variable having a compile-time constant value.  These will eventually be renamed from Variable to Constant.